spread_tests/scripts/kernos_daemon

```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+
+# This script wraps kernos to run commands as the remote daemon
+
+set -e
+
+KERNOS_RUNTIME_DIR=$("${SCRIPTS_DIR}/get_kernos_runtime_dir")
+
+export PEBBLE="/"
```

Comment:
You'll most likely need to pass in `KERNOS=1` to envp for `boot-firmware` to succeed (see KO008).

```suggestion
export PEBBLE="/"
export KERNOS=1
```